includes/definitions/discovery/exablaze-fusion.yaml

```diff
@@ -0,0 +1,58 @@
+mib: EXALINK-FUSION-MIB
+modules:
+    sensors:
+        pre-cache:
```

Comment:
You can drop the `pre-cache` stuff here. It's only needed if you need access to other snmp data.